enable/example_application.py

```diff
@@ -12,64 +12,17 @@
 ===========================
 
 This module provides a simple Pyface application that can be used by examples
-in places where a DemoFrame is insufficient.
+in places where a DemoFrame is insufficient. Note this has been moved to
+sit in enable/examples.  This module is kept for backwards compatibility.
```

Comment:
Let's go with the following for the module docstring - and the warning message as well.

```
Importing from this module is deprecated. Please use `enable.examples.example_application`
instead. This backwards compatibility stub will be removed in Enable 6.0.
```

We need an issue to track the removal of these stubs in Enable 6.0

Reply:
let's use the above template for the module docstring and warning message in the other stub modules as well.

Reply:
Should we say `Please use `enable.examples.example_application` instead.` if we are making the modules private?  Theoretically the suggested change is "stop doing that" as opposed to "update your imports"

Reply:
Ahh. You're right. Let's update it to

```
This module is meant for internal use only and it is not meant for use in library code.
Importing from this module is deprecated and it will be removed in Enable 6.0.
We highly recommend that you update your code and vendorize what is necessary.
```